web/src/reducers/budget.js

```diff
@@ -62,6 +63,18 @@ const initialState = years => ({
   years
 });
 
+const budgetInput = (type, value = v => +v, target = null) => ({
+  type,
+  value,
+  target: target || type
+});
+
+const budgetInputs = [
+  budgetInput('statePersonnel', year => +year.amt * +year.perc / 100),
+  budgetInput('contractorResources', undefined, 'contractors'),
```

Comment:
Maybe we should have a cleanup issue to change this to `contractors` on the server side too, make it consistent with the other two?

Reply:
agreed